.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
usually `.vscode` and any other ide settings are not commited

Reply:
yeah, I know that. but this repository is not interesting to anyone, but me, so I decided to do it for my convenience.